lib/emitter.js

```diff
@@ -9,7 +9,9 @@ exports.fragment = function (doc) {
     return '';
   }
 
-  return doc[0].contents.map(emit).join('');
+  return doc.reduce(function (arr, para) {
```

Comment:
Haha this is a lot of iterations and copies of the array... Might be simplest to just have a loop at this point?